addons/mail/static/src/rtc/call_invitations.js

```diff
@@ -1,10 +1,10 @@
 /** @odoo-module **/
 
-import { useStore } from "@mail/core/messaging_hook";
-import { Component } from "@odoo/owl";
+import { CallInvitation } from "@mail/rtc/call_invitation";
```

Comment:
Note: if we decide to make a separate folder for `rtc`, the complete `rtc` feature should be contained within this folder, and not have parts outside of it.
Either move everything related inside (with extra override/different store for `rtc` features), or don't make a separate folder.

Reply:
Actually rtc was not moved, just some parts with members.
This comment can be considered for another PR.